README.md

```diff
@@ -1360,6 +1360,7 @@ See **[Ticketing](#ticketing)**
 - [Babelfish](https://github.com/bblfsh/bblfshd) - Self-hosted server for source code parsing. It can parse any file, in any supported language, extract an Abstract Syntax Tree from it, and convert it to a Universal Abstract Syntax Tree which can enable further analysis and transformation. `GPL-3.0` `Go`
 - [Code-Server](https://coder.com/) - Visual Studio Code in the browser, hosted on a remote server. ([Source Code](https://github.com/cdr/code-server)) `MIT` `Nodejs/Docker`
 - [Eclipse Che](http://www.eclipse.org/che/) - Open source workspace server and cloud IDE. ([Source Code](https://github.com/eclipse/che)) `EPL-1.0` `Docker/Java`
+- [Gitpod](https://gitpod.io/) - Open source vscode-like and vscode extension compatible cloud IDE provided as upstream [theia](https://github.com/eclipse-theia/theia) distribution with automated self-hosting deployment for your infrastructure. ([Demo](https://gitpod.io/#https://github.com/awesome-selfhosted/awesome-selfhosted), [Source Code](https://github.com/gitpod-io/self-hosted)) `EPL-2.0` `Typescript/Go/Docker`
```

Comment:
This description makes no sense... :) Please reuse the description from https://www.gitpod.io/ (_Online IDE for GitHub and GitLab_). The inline link to theia was what was causing CI failures

Reply:
Sorry for the delay

Reply:
Resolved in [https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2104/commits/98aa8320b3dd577c16cab17ddee419f422033697 ](https://github.com/awesome-selfhosted/awesome-selfhosted/pull/2104/commits/98aa8320b3dd577c16cab17ddee419f422033697)